python/plugins/phidata/composio_phidata/toolset.py

```diff
@@ -28,39 +28,60 @@ def _wrap_tool(
         self,
         schema: t.Dict,
         entity_id: t.Optional[str] = None,
-    ) -> Function:
+    ) -> Toolkit:
         """
-        Wrap composio tool as Phidata `Function` object.
+        Wrap composio tool as Phidata `Toolkit` object.
         """
         name = schema["name"]
         description = schema["description"]
         parameters = schema["parameters"]
-
+        
+        # Create a new Toolkit instance
+        toolkit = Toolkit(name=name)
+        
+        @validate_call
         def function(**kwargs: t.Any) -> str:
```

Comment:
Consider using a more specific type hint than `t.Any` for the kwargs parameter. Since we have the schema available, we could potentially create a TypedDict based on the parameters schema to provide better type safety.